tests/v2/integration/cli/test_cli.py

```diff
@@ -34,9 +34,9 @@ def test_otx_cli(task: str, tmp_dir_path: Path) -> None:
         "--work_dir", str(tmp_dir_path),
         "--data.task", task,
         "--data.train_data_roots", TASK_CONFIGURATION[task]["train_data_roots"],
+        "--data.val_data_roots", TASK_CONFIGURATION[task]["val_data_roots"],
         "--max_epochs", "1",
         "--seed", "1234",
-        "--deterministic", "True",
```

Comment:
Specific reason for this?

Reply:
I found a simple error (maybe it's only happening locally on my end).
```
RuntimeError: cumsum_cuda_kernel does not have a deterministic implementation, but you set 'torch.use_deterministic_algorithms(True)'. You can turn off determinism just for this operation, or 
you can use the 'warn_only=True' option, if that's acceptable for your application. You can also file an issue at https://github.com/pytorch/pytorch/issues to help us prioritize adding 
deterministic support for this operation.
```

With `export CUBLAS_WORKSPACE_CONFIG=:16:8`, this also works fine with no errors. I haven't looked closely yet, maybe I should leave this as True and look into the issue a bit more?
Or maybe I can revisit this at a later date.

Reply:
https://github.com/openvinotoolkit/training_extensions/blob/268aed7a92b829d68ff1eb9093cc8f180c1d4dfa/src/otx/algorithms/visual_prompting/tasks/train.py#L68
I think it can be a simple solution, what do you think?

Reply:
I added Sungchul's suggestion and reverted this code.